### PR TITLE
Add docs for ALTER SEQUENCE and sequence functions

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

Closes #16648 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16649
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-alter-sequence.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+title: ALTER SEQUENCE
+summary: ALTER SEQUENCE 在 TiDB 中的使用概况
```
Comment (qiancai) [inline sql-statements/sql-statement-alter-sequence.md:3]:
```suggestion
summary: 介绍 ALTER SEQUENCE 在 TiDB 中的使用概况。
```

File: sql-statements/sql-statement-alter-sequence.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+```
++-------------+
+| NEXTVAL(s1) |
++-------------+
+|           2 |
++-------------+
+1 row in set (0.00 sec)
+```
+
+将序列的步长更改为 `2`：
```
Comment (qiancai) [inline sql-statements/sql-statement-alter-sequence.md:125]:
```suggestion
将该序列的步长更改为 `2`：
```

File: functions-and-operators/sequence-functions.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: 序列函数
+summary: This document introduces sequence functions supported in TiDB. 了解 TiDB 中的序列函数。
```
Comment (qiancai) [inline functions-and-operators/sequence-functions.md:3]:
```suggestion
summary: 了解 TiDB 中的序列函数。
```

File: functions-and-operators/sequence-functions.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+# 序列函数
+
+TiDB 中的序列函数用于返回或设置使用 [`CREATE SEQUENCE`](/sql-statements/sql-statement-create-sequence.md) 语句创建的序列对象的值。
+
+| 函数名称 | 功能描述 |
+| :-------- | :-------------------------- |
+| `NEXTVAL()` 或 `NEXT VALUE FOR` | 返回序列的下一个值 |
+| `SETVAL()` | 设置序列的当前值 |
+| `LASTVAL()` | 返回序列中最后一个使用过的值 |
```
Comment (qiancai) [inline functions-and-operators/sequence-functions.md:14]:
```suggestion
| `LASTVAL()` | 返回序列中最近一个使用过的值 |
```

File: sql-statements/sql-statement-alter-sequence.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+示例如下：
+
+```
+1, 3, 5, ...            // 序列遵循起始为 1、步长为 2 的等差关系。
+SELECT SETVAL(seq, 6)   // 设置序列的当前值为 6。
+7, 9, 11, ...           // 后续产生值仍会遵循这个等差关系。
+```
+
+在 `CYCLE` 模式下，序列的起始值第一轮为 `start`，后续轮次将会是 `MinValue` (increment > 0) 或 `MaxValue` (increment < 0)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-alter-sequence.md:208]:
```suggestion
在 `CYCLE` 模式下，序列的起始值第一轮为 `START`，后续轮次将会是 `MinValue` (INCREMENT > 0) 或 `MaxValue` (INCREMENT < 0)。
```

File: sql-statements/sql-statement-create-sequence.md [hunk 6]
```diff
@@ -97,12 +98,12 @@ CREATE [TEMPORARY] SEQUENCE [IF NOT EXISTS] sequence_name
...
++ 使用 `NEXTVAL()` 函数获取序列对象的下一个值。
 
     {{< copyable "sql" >}}
 
     ```sql
-    SELECT nextval(seq);
+    SELECT NEXTVAL(seq);
     ```
 
     ```
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-create-sequence.md:109]: L111:
```
    | NEXTVAL(seq) |
```

File: sql-statements/sql-statement-create-sequence.md [hunk 7]
```diff
@@ -306,13 +307,15 @@ CREATE [TEMPORARY] SEQUENCE [IF NOT EXISTS] sequence_name
 
 ```
 1, 3, 5, ...            // 序列遵循起始为 1、步长为 2 的等差关系。
-select setval(seq, 6)   // 设置序列的当前值为 6。
+select SETVAL(seq, 6)   // 设置序列的当前值为 6。
 7, 9, 11, ...           // 后续产生值仍会遵循这个等差关系。
 ```
 
 在 `CYCLE` 模式下，序列的起始值第一轮为 `start`，后续轮次将会是 `MinValue` (increment > 0) 或 `MaxValue` (increment < 0)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-create-sequence.md:314]:
```suggestion
在 `CYCLE` 模式下，序列的起始值第一轮为 `START`，后续轮次将会是 `MinValue` (INCREMENT > 0) 或 `MaxValue` (INCREMENT < 0)。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: sql-statements/sql-statement-create-sequence.md (hunk 6 above)
Comment (qiancai) [reply sql-statements/sql-statement-create-sequence.md:109]: updated in https://github.com/pingcap/docs-cn/pull/17220/commits/12d2ef48b19c0367986a8dfb477a72831a6a5dcb

**Pull request merged**